MobileWallet/TariLib/Wrappers/Utils/ConnectionMonitor.swift

```diff
@@ -162,6 +162,7 @@ class ConnectionMonitor {
 
         TariEventBus.onMainThread(self, eventType: .torConnectionFailed) { [weak self] (_) in
             guard let self = self else { return }
+            Tracker.shared.track(eventWithCategory: "Transaction", action: "User's transaction fails - Tor issue", name: "Transaction Failed - Tor Issue")
```

Comment:
This happens sometimes when a user is not trying to send a tx so it'll cause incorrect reporting. Move to `handleSendingComplete()` inside `SendingTariViewController`. When an error is received there check the tor connection state. If it's offline then report this